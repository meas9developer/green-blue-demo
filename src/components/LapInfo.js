import React from 'react'
import { View, StyleSheet, Text } from 'react-native'
import formatTime from 'minutes-seconds-milliseconds'
import PropTypes from 'prop-types'

const styles = StyleSheet.create({
  item: {
    fontSize: 30
  }
})

const LapInfo = ({lap}) => {
  return (
    <View>
      <Text style={styles.item}>{formatTime(lap)}</Text>
    </View>
  )
}


LapInfo.propTypes = {
  lap: PropTypes.number
}
LapInfo.defaultProps = {}

export default LapInfo
