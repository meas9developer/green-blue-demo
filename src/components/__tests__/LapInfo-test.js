import React from 'react'
import LapInfo from './../LapInfo'
import renderer from 'react-test-renderer'

it('renders without crashing', () => {
  const rendered = renderer.create(<LapInfo />).toJSON()
  expect(rendered).toBeTruthy()
  expect(rendered).toMatchSnapshot()
});
