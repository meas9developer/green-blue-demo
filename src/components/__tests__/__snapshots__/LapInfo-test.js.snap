// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 30,
      }
    }
  >
    00:00.00
  </Text>
</View>
`;
